docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.color-band.scss

```diff
@@ -1,6 +1,8 @@
 .hb-color-band {
-  text-align: center;
   position: relative;
+  padding: hb-calculate-rems(12px) hb-calculate-rems(24px) hb-calculate-rems(14px);
```

Comment:
I think that the bottom padding should changed to 2px, because it gets added to the bottom padding/margin of the last element, making the bottom spacing larger than expected.

Title only:
<img width="998" alt="Screenshot 2024-01-30 at 18 19 43" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/2353a93d-5252-45f0-adc8-0d8192cdeea1">

Title + additional text:
<img width="988" alt="Screenshot 2024-01-30 at 18 20 01" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/0d5e127e-50ab-4f52-a38e-45a13cf61cb4">

Reply:
Ugh, I am sorry - I had fixed that but for some reason didn't push it. Pushing that now!